webapp/src/Controller/API/ContestController.php

```diff
@@ -363,75 +363,74 @@ public function changeTimesAction(Request $request, string $cid): Response
         $now      = (int)Utils::now();
         $changed  = false;
         if (!$request->request->has('id')) {
-            $response = new JsonResponse('Missing "id" in request.', Response::HTTP_BAD_REQUEST);
+            return new JsonResponse('Missing "id" in request.', Response::HTTP_BAD_REQUEST);
         } elseif (!$request->request->has('start_time') && !$request->request->has('scoreboard_thaw_time')) {
```

Comment:
Nitpick: I'd also change these `elseif`'s to `if`'s. Otherwise it seems at a glance that there's a more complicated control flow, than simply a list of "check for this error condition and throw an error".